src/thermostat.js

```diff
@@ -173,7 +173,6 @@ export default function createThermostat({ Service, Characteristic }) {
         fn(...args)
         this.startDisconnectTimeout()
       }, (err) => {
-        err.isError = true
```

Comment:
I'm not sure if this fix will work as expected. The `isError` property is used for error handling in the `get` and `set` handlers of a characteristic.
I think the reason why there occurs a unhandled promise rejection is the removal of this line: https://github.com/maxnowack/homebridge-eq3ble/pull/3/commits/0b646cfb152299b527967418e364342bfb1b5916#diff-ec4f916e4e15cb242a8f575de564c2f4L60